L06-annotations/src/main/java/ru/denis_ch/my_test_framework/TestExe.java

```diff
@@ -0,0 +1,105 @@
+package ru.denis_ch.my_test_framework;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.HashSet;
+import java.util.Set;
+
+public class TestExe {
+    private int allCntBefore;
+    private int allCntTest;
+    private int allCntAfter;
+
+    private int errCntBefore;
+    private int errCntTest;
+    private int errCntAfter;
+
+
+    public void do_test(String className) {
```

Comment:
В java имена методов принято писать в `camelCase`:
```suggestion
    public void doTest(String className) {
```

Reply:
+

Reply:
+